app/assets/javascripts/govuk_publishing_components/components/step-by-step-nav.js

```diff
@@ -78,6 +81,14 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
     this.$module.insertBefore(showAll, steps)
     this.$module.showOrHideAllButton = this.$module.querySelectorAll('.js-step-controls-button')[0]
+
+    // if GA4 is enabled, set attributes on 'show all sections' for tracking using ga4-event-tracker
+    if (this.$module.isGa4Enabled) {
+      var indexTotal = this.$module.totalSteps
+      var showAllAttributesGa4 = { event_name: 'select_content', type: 'step by step', index: 0, index_total: indexTotal }
```

Comment:
Nitpick - might as well put `this.$module.totalSteps` directly into the object rather than declaring it separately.